includes/admin/payments/refunds.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Refund Details Sections
+ *
+ * @package     EDD
+ * @subpackage  Admin/Orders
+ * @copyright   Copyright (c) 2018, Easy Digital Downloads, LLC
```

Comment:
Since this is a new file, should be 2020. Wondering if it should be Sandhills instead of EDD?